website/versioned_docs/version-0.62/getting-started.md

```diff
@@ -0,0 +1,115 @@
+---
+id: version-0.62-getting-started
+title: Get Started with Windows
+original_id: getting-started
+---
+
+This guide will help you get started on setting up your very first React Native for Windows app.
+
+Make sure you have installed all of the [development dependencies](rnw-dependencies.md).
+
+If you're looking to build directly from the repo or you want to make contributions to the react-native-windows, check out the guide for [building the react-native-windows repo](building-rnw.md).
+
+For information around how to set up React Native, see the [React Native Getting Started Guide](https://reactnative.dev/docs/getting-started).
+
+## Install React Native for Windows
+
+Remember to call `react-native init` from the place you want your project directory to live.
+
+```
+npx react-native init <projectName> --version ^0.62.2
+```
+>To create TypeScript template, run `npx react-native init <projectName> --version ^0.62.2 --template react-native-template-typescript`.<br><br>
+> If you've installed react native globally in the past, via `npm install -g react-native`, and are having issues with the new instructions, try running:<br>
+> `npx --ignore-existing react-native init <projectName> --template react-native@^0.62.2` instead.
+
+### Navigate into this newly created directory
+
+Once your project has been initialized, React Native will have created a new sub directory where all your generated files live.
+
+```
+cd <projectName>
+```
+
+### Install the Windows extension
+
+Lastly, install the React Native for Windows packages.
+
+```
+npx react-native-windows-init --overwrite
+```
+
+> The --overwrite flag is a temporary measure that ensures the correct files are copied to metro.config.js for the metro bundler to work with Windows. If you are starting a new app, this should have no impact. If you are adding Windows to your existing app and you have modified the metro.config.js file, please back up your changes, run the command and copy over to take effect. We are tracking [this issue here](https://github.com/microsoft/react-native-windows/issues/4698).
+
+Here are the options that `react-native-windows-init` takes:
+```none
+Options:
+  --help       Show help                                               [boolean]
+  --version    The version of react-native-windows to use.              [string]
+  --namespace  The native project namespace.                            [string]
+  --verbose    Enables logging.                                        [boolean]
+  --language   Which language the app is written in.
```

Comment:
@harinikmsft @kikisaints should we mention experimental / work-in-progress options like useWinUI3 or the use experimental nuget option?

Reply:
Not a bad idea. for useWinUI3 - would be good to cross link to the guide for WinUI3 for more context. Do we have similar context for nuget option?

Reply:
I remember ACoates was wanting to leave NuGet out, since we don't want external folks to use it until its in a better state.

Reply:
then let's just add the `--useWinUI3     Targets WinUI 3.0 Alpha instead of UWP XAML                 [boolean]`

Reply:
This is also pulled directly from the `react-native-windows-init` README (see https://www.npmjs.com/package/react-native-windows-init). We should make sure to update that, and possibly consider linking there so the two stay in sync.

Reply:
Here's where that lives: https://github.com/microsoft/react-native-windows/tree/master/packages/react-native-windows-init 
(It needs to live there to show up as package docs)

Reply:
so just to clarify, do you want to add this: `--useWinUI3 Targets WinUI 3.0 Alpha instead of UWP XAML [boolean]` to that table in our getting started guide, or should I wait for the nuget and readme to update first?

Reply:
@kikisaints    yup yup, the nuget stuff is independent of the winui3 support

Reply:
Done, thanks!

Reply:
can you add an inline link to the WinUI 3 doc as well so people know what the difference is?